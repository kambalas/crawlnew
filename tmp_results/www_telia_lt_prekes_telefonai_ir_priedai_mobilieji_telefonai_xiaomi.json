[
  {
    "name": "Xiaomi 15 Ultra 5G",
    "price": "1258",
    "error": false
  },
  {
    "name": "Xiaomi 15 5G",
    "price": "799",
    "error": false
  },
  {
    "name": "Xiaomi 14T",
    "price": "399",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 4G",
    "price": "219",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 5G",
    "price": "219",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 Pro 5G",
    "price": "289",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 Pro+ 5G",
    "price": "499",
    "error": false
  },
  {
    "name": "Xiaomi Redmi 14C",
    "price": "99",
    "error": false
  },
  {
    "name": "Xiaomi 14T Pro",
    "price": "689",
    "error": false
  },
  {
    "name": "Xiaomi Redmi 13 4G",
    "price": "157",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 Pro+ 5G",
    "price": "289",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 Pro 5G",
    "price": "199",
    "error": false
  }
]