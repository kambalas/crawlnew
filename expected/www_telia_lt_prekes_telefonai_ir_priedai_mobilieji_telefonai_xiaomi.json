[
  {
    "name": "Xiaomi 15 Ultra 5G 512 gb įvairiaspalvė",
    "price": "1258",
    "error": false
  },
  {
    "name": "Xiaomi 15 5G 512 gb juoda",
    "price": "799",
    "error": false
  },
  {
    "name": "Xiaomi 14T 256 gb juoda",
    "price": "399",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 4G 128 gb juoda",
    "price": "219",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 5G 128 gb juoda",
    "price": "219",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 Pro 5G 256 gb juoda",
    "price": "289",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 Pro+ 5G 512 gb juoda",
    "price": "499",
    "error": false
  },
  {
    "name": "Xiaomi Redmi 14C 128 gb juoda",
    "price": "99",
    "error": false
  },
  {
    "name": "Xiaomi 14T Pro 512 gb juoda",
    "price": "689",
    "error": false
  },
  {
    "name": "Xiaomi Redmi 13 4G 128 gb juoda",
    "price": "157",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 Pro+ 5G 512 gb juoda",
    "price": "289",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 Pro 5G 256 gb juoda",
    "price": "199",
    "error": false
  }
]